support/browserify.js

```diff
@@ -27,7 +27,10 @@ function build(fn){
     entries: [ path ],
     insertGlobalVars: { global: glob },
     standalone: 'eio'
-  }).bundle();
+  })
+  .exclude('bufferutil')
+  .exclude('utf-8-validate')
+  .bundle();
```

Comment:
To comply with https://github.com/websockets/ws/commit/49b11093e9a009e5305dcde7003d3a896b2811dc